docs/Building.md

```diff
@@ -37,3 +37,17 @@ $ sbt server/docker
 ```
 
 This will build and tag a Docker image with a name like `broadinstitute/cromwell:<VERSION>-SNAP`.
+
+###IntelliJ gotchas
+
+If you develop Cromwell using IntelliJ IDEA, you may the following error message on startup:
```

Comment:
Maybe specify "when starting SBT" to make clear that it's not Cromwell itself that's missing adequate memory